### PR TITLE
Merge 2.x

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

**Review by VincentLanglet — APPROVED** [review]

File: src/Seo/SeoPage.php [hunk 1]
```diff
@@ -135,16 +135,16 @@ public function removeMeta($type, $name)
...
-    public function setMetas(array $metadatas)
+    public function setMetas(array $metas)
     {
         $this->metas = [];
 
-        foreach ($metadatas as $type => $metas) {
-            if (!\is_array($metas)) {
+        foreach ($metas as $type => $metaItem) {
+            if (!\is_array($metaItem)) {
                 throw new \RuntimeException('$metas must be an array');
```
Comment (phansys) [inline src/Seo/SeoPage.php:144]: This message seems outdated.

**Review by phansys — APPROVED** [review]

File: src/Seo/SeoPage.php (hunk 1 above)
Comment (core23) [reply src/Seo/SeoPage.php:144]: I'll check this later. I'll try to harden the 3.x branch in the next days, so we can release a new major version soon.

File: composer.json [hunk 2]
```diff
@@ -38,11 +38,18 @@
     "require-dev": {
         "knplabs/knp-menu": "^3.0",
         "knplabs/knp-menu-bundle": "^3.0",
+        "phpstan/extension-installer": "^1.0",
+        "phpstan/phpstan": "^0.12.84",
+        "phpstan/phpstan-phpunit": "^0.12.16",
+        "psalm/plugin-phpunit": "^0.15",
+        "sonata-project/doctrine-extensions": "^1.10.1",
```
Comment (VincentLanglet) [inline composer.json:45]: sonata-project/doctrine-extensions and sonata-project/form-extensions are now in both require and require-dev @core23

Comment (core23) [reply composer.json:45]: Oh, you're right. I'll fix this

**Pull request merged**